source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -3241,13 +3241,27 @@ void Graph::graphButtons()
 
     // Create two windows using splitter
     float paneWidth = (leftPaneWidth - 2.0f);
-    ImGui::BeginChild("Selection", ImVec2(paneWidth, 0));
+    
+    float aspectRatio = _renderer->getPixelRatio();
+    ImVec2 screenSize = ImVec2(paneWidth, paneWidth / aspectRatio);
+
+    ImVec2 mousePos = ImGui::GetMousePos();
+    ImVec2 tempWindowPos = ImGui::GetCursorPos();
+    bool cursorInRenderView = mousePos.x > tempWindowPos.x && mousePos.x < (tempWindowPos.x + screenSize.x) &&
+                              mousePos.y > tempWindowPos.y && mousePos.y < (tempWindowPos.y + screenSize.y);
+    
+    ImGuiWindowFlags windowFlags = 0;
+
+    if (cursorInRenderView)
+    {
+        windowFlags |= ImGuiWindowFlags_NoScrollWithMouse;
```

Comment:
@HudsonHN Does this affect the ability to scroll and zoom inside the render view window?